src/applications/vaos/tests/new-appointment/components/VAFacilityPage/index.unit.spec.js

```diff
@@ -264,6 +265,14 @@ describe('VAOS Page: VAFacilityPage', () => {
         store,
       });
 
+      // Radio buttons only show up after all the data is loaded, which
+      // should mean all page rendering is finished
+      await screen.findAllByRole('radio');
+
+      // This functionality is not possible given current implementation of
+      // FORM_PAGE_FACILITY_V2_OPEN_SUCCEEDED in the reducer.js. On lines
+      // 324-326, vaFacility is cleared when multiple facilities offers the
+      // requested type of care.
```

Comment:
@JunTaoLuo, How about reverting the FORM_PAGE_FACILITY_V2_OPEN_SUCCEEDED logic prior to last week's change. Then replacing lines 340-344 with
 ```
 const { data, schema } = setupFormData(
    (newData = {
      ...newData,
      isSingleVaFacility: typeOfCareFacilities.length === 1,
    }),
    newSchema,
    action.uiSchema,
  );
```

Reply:
Yup this worked! Thanks!